src/main/scala/nl/knaw/dans/easy/stage/fileitem/EasyFileMetadata.scala

```diff
@@ -15,17 +15,18 @@
  */
 package nl.knaw.dans.easy.stage.fileitem
 
-import scala.xml.Elem
+import scala.util.Try
 
 object EasyFileMetadata {
-  def apply(s: FileItemSettings): Elem =
+  def apply(s: FileItemSettings): Try[String] = Try {
       <fimd:file-item-md xmlns:fimd="http://easy.dans.knaw.nl/easy/file-item-md/" version="0.1" >
         <name>{s.pathInDataset.get.getName}</name>
         <path>{s.pathInDataset.get}</path>
         <mimeType>{s.format.getOrElse("application/octet-stream")}</mimeType>
-        <size>{s.size.get}</size>
+        <size>{s.size.getOrElse(0)}</size>
```

Comment:
Size zero doesn't seem to cause a problem in the round trip test. To show the real value would require a http-header request with the specified url, if that provides the length.